crates/l3/common/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "common"
```

Comment:
terrible package name

Reply:
imagine the package is published on crate.io. This would be it's name.

We can find a more descriptive name.

Reply:
moreover it looks like it is mostly used in `appchain-core-contract-client`. So maybe it should just live there as `error` and `utils` modules

Reply:
noted.
I can rename it to appchain-utils but i believe this will be required as methods invoke_contract and call_contract will be used for another module like verifier contract to be added next hence kept it as a common/utils package.